src/app/base/components/bricks-integrator/bricks-integrator.component.ts

```diff
@@ -172,7 +176,10 @@ export class BricksIntegratorComponent implements OnInit, OnDestroy {
         if (this.config.extendedIntegrator) {
           finalData = data.data.map(e => {
             const c = jsonCopy(e);
```

Comment:
What does "e" and "c" mean in this context? I could maybe search it up in the broader context but would prefer to have meaningfull names here :)